### ISSUE TITLE
DVS- Dover Spooler

### ISSUE THREAD

**Issue by eswenson1** [PDP-10/its#242]
Do we need/want it?  Who has a DOVER (it was a laser printer, if I remember correctly) these days?

DEVICE; JOBDEV DVS
SYSEN1; DVRDEV 115

**Comment by larsbrinkhoff** [comment]
Only when we can emulate such a printer do we need the device.

**Comment by larsbrinkhoff** [comment]
I have a theory that the Dover was first connected to MC, and later moved to another building and made accessible through a MINITS box via Chaosnet.

This DVRDEV looks like it's from the MC era.  "It run only on the MC machine; other machines use device MCDVR:"  In addition to DVR: providing a status listing, it can also run as a Chaosnet service with a host of contact names: DVR, DOV, DVS, DOS, DVQ, and DOQ.

Some code was originally copied from XGPDEV.

**Comment by dabridgham** [comment]
I remember the Dover being connected to an Alto on the 3Mb/s Ethernet.  I'm not sure how the Chaosnet world got to it but through a MINITS box makes seems likely.

**Comment by larsbrinkhoff** [comment]
Right, an Alto running a program called Spruce.  So for my theory to make sense, that Alto would (only?) have been accessible from MC for a period.

The MINTS part of the theory comes from the fact that there is a file called MITS.S; DOVER 1 titled "DOVER server (CHAOS/PUP)".

**Comment by larsbrinkhoff** [comment]
@dabridgham, I have now scrutinized both the old IOELEV code and MINITS.  It seems they both support gatewaying printing from Chaosnet to PUP on Ethernet.

**Comment by larsbrinkhoff** [comment]
The ContrAlto emulator is capable of running Spruce and generates PDFs.
